src/experimental/safe_html_transform/safe_html.py

```diff
@@ -0,0 +1,2630 @@
+import logging
```

Comment:
@prakharjoshi this pull requests includes an unregistered transform, this should be removed.